concent_api/core/management/commands/concent_adapt_data_to_current_version.py

```diff
@@ -0,0 +1,29 @@
+from logging import getLogger
+
+from django.core.management.base import BaseCommand
+from typing import Any
```

Comment:
`typing` import should be after `logging`.

Reply:
Done